test/dlc_tests/ec2/pytorch/training/test_pytorch_training.py

```diff
@@ -159,7 +160,15 @@ def test_pytorch_linear_regression_gpu(
         pytest.skip(
             f"Image {pytorch_training} is incompatible with instance type {ec2_instance_type}"
         )
-    execute_ec2_training_test(ec2_connection, pytorch_training, PT_REGRESSION_CMD)
+    _, image_framework_version = get_framework_and_version_from_tag(pytorch_training)
+    image_cuda_version = get_cuda_version_from_tag(pytorch_training)
+    if (
+        Version(image_framework_version) in SpecifierSet(">=2.0")
+        and image_cuda_version == "cu121"
```

Comment:
Please compare CUDA versions as done here:

https://github.com/aws/deep-learning-containers/blob/163e4ed6b06dcdb81ec2c10e0065abeb3e69880a/test/sagemaker_tests/pytorch/training/integration/sagemaker/test_smdataparallel.py#L41-L44

Also, the test should work for CUDA versions >=12.1, and not just 12.1, right?

Reply:
made code change